.vortex/tests/phpunit/Functional/ComposerInstallerTest.php

```
@@ -0,0 +1,98 @@
+ * Vortex relies on 'composer/installers' alone (without an installer-extender
+ * plugin) to place 'drupal-library' packages into 'web/libraries/{$name}'.
+ * This guards that the install path keeps working using the template's own
+ * installer configuration.
+ */
+class ComposerInstallerTest extends FunctionalTestCase {
+
+  protected function setUp(): void {
+    // Only locations are required: the test reads the template 'composer.json'
+    // and runs Composer in a temporary directory. No SUT or Docker stack is
+    // involved, so the heavy parent setUp is intentionally not called.
+    self::locationsInit(File::cwd() . '/../..');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\AlexSkrypnyk\File\File' in method 'setUp'. (undefined)

(StaticAccess)